### ISSUE TITLE
Add Save Keyboard Shortcut

### ISSUE THREAD

**Issue by jauyong** [GoogleForCreators/web-stories-wp#1056]
## Feature Description

As an editor I want a keyboard shortcut for saving
---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria

1. I should be able to use keyboard shortcuts for saving. Control + S, CMD + S...

**Comment by miina** [comment]
Closing as this is already implemented.
